python/private/pypi/pip_compile.bzl

```diff
@@ -53,8 +53,8 @@ def pip_compile(
 
     Args:
         name: base name for generated targets, typically "requirements".
-        src: file containing inputs to dependency resolution. If not specified,
-            defaults to `pyproject.toml`. Supported formats are:
+        srcs: a list of files containing inputs to dependency resolution. If not specified,
```

Comment:
If you want to support srcs list, you should add it as a new arg instead of changing the existing arg. As you've seen, you had to modify all of the examples to make this work, which means that all of the users would have to do this change as well.